dashboard/dashboard/templates/racetrack/administration.html

```diff
@@ -46,9 +46,16 @@ <h5 class="card-title">Infrastructure Targets</h5>
             Available Infrastructure Targets:
             {% if infrastructure_targets|length > 0 %}
                 <ul>
-                {% for infrastructure_target, plugin_manifest in infrastructure_targets.items %}
+                {% for infrastructure, instances in infrastructure_instances %}
+                <span>{{ infrastructure }}</span>
                     <li>
```

Comment:
Sorry to say that, but I guess it should be 
```suggestion
                    <li><span>{{ infrastructure }}</span>
```
cause it still looks wrong in Chrome:
![Screenshot from 2023-01-19 13-10-17](https://user-images.githubusercontent.com/45496492/213439943-6484c9a7-6744-4c13-9fa6-b24ffb298a0b.png)